src/components/Footer.jsx

```diff
@@ -1,9 +1,11 @@
 import { Row, Col } from "antd";
 import "./footer.css";
+import PropTypes from "prop-types";
 
-const Footer = () => {
+const Footer = ({ sticky }) => {
+  const footerClass = `footer-container ${sticky ? "sticky" : ""}`;
```

Comment:
If a page doesn't have enough content to fill the view height, then the footer kind of tucks up under the last element  as you can see here: 

<img width="1512" alt="non-sticky-footer" src="https://github.com/rknizzle/propermesh-view/assets/111093132/69ac7b2e-0862-413c-b7cf-7337f75d7445">

By adding the sticky class, it sticks it to the bottom of the page: 

<img width="1512" alt="sticky-footer" src="https://github.com/rknizzle/propermesh-view/assets/111093132/8d564258-a62e-4393-b37c-722eec793599">

We would keep the sticky class on the login/register page, but eventually remove it from the app page when its completed. 

The landing page has enough content to push the footer to the bottom naturally

Reply:
Gotcha thanks for the screenshot! Does it cause a problem when the footer is made sticky on the pages that naturally push it to the bottom anyway? If not we could just make it always sticky so that we can get rid of this option

Reply:
Yeah it causes the footer to always be visible on the screen. We could do that if its something you'd prefer, I'm personally against it. 

I'll show you the difference when we get together and you can make the final call!

Reply:
I think another option could be to only have the footer on the landing page and not on any other pages. What do you think about that?